src/SonarQube.Client/ISonarQubeService.cs

```diff
@@ -53,6 +53,8 @@ public interface ISonarQubeService
         Task<IList<SonarQubeRule>> GetRulesAsync(bool isActive, string qualityProfileKey,
             CancellationToken token);
 
+        Task<SonarQubeRule> GetRuleByKeyAsync(string ruleKey, CancellationToken token);
```

Comment:
Missing doc comment (I know other existing methods are missing doc comments, but we should certainly be adding them for new methods).

Is the `ruleKey` the "composite" rule key i.e. in the form `[repo id]:[rule id]` e.g. `cpp:S123`?